CHANGELOG.md

```diff
@@ -1,444 +1,624 @@
 # Changelog
 
 ## 0.17.3-dev
- - [#565](https://github.com/tag1consulting/goose/pull/565) add `--accept-invalid-certs` to skip validation of https certificates
- - [#568](https://github.com/tag1consulting/goose/pull/568) don't panic when truncating non utf-8 string
- - [#574](https://github.com/tag1consulting/goose/pull/574) update [`http`](https://docs.rs/http), [`itertools`](https://docs.rs/itertools) [`nix`](https://docs.rs/nix), [`rustls`](https://docs.rs/rustls/), and [`serial_test`](https://docs.rs/serial_test)
- - [#575](https://github.com/tag1consulting/goose/pull/575) add test coverage for sessions and cookies, revert [#557](https://github.com/tag1consulting/goose/pull/557) to avoid sharing the CookieJar between all users
+
+- [#565](https://github.com/tag1consulting/goose/pull/565) add `--accept-invalid-certs` to skip validation of https
+  certificates
+- [#568](https://github.com/tag1consulting/goose/pull/568) don't panic when truncating non utf-8 string
+- [#574](https://github.com/tag1consulting/goose/pull/574)
+  update [`http`](https://docs.rs/http), [`itertools`](https://docs.rs/itertools) [`nix`](https://docs.rs/nix), [`rustls`](https://docs.rs/rustls/),
+  and [`serial_test`](https://docs.rs/serial_test)
+- [#575](https://github.com/tag1consulting/goose/pull/575) add test coverage for sessions and cookies,
+  revert [#557](https://github.com/tag1consulting/goose/pull/557) to avoid sharing the CookieJar between all users
+- [#600](https://github.com/tag1consulting/goose/pull/600) Refactor reports/metrics, add JSON and markdown report
```

Comment:
I think your editor is reformatting this entire file, I'd prefer to not make unrelated changes like this.

Reply:
Should be fixed.

Reply:
Perfect, thank you! I've started reviewing, will aim to get it merged quickly (probably tomorrow at this rate).